car-cards.js

```diff
@@ -0,0 +1,24 @@
+//AUGMENTED IIFE FOR CHANGING THE BORDER / BACKGROUND WHEN CAR IS CLICKED
+var CarLot = (function(carlot) {
+  var cardsInDOM = document.getElementsByClassName("cards"); // cardsInDOM is an array of inventory divs, class given to div that holds car elements
+  var textInput = document.getElementById("text-input"); // text input in nav bar
+  var cardSelected;
+
+
+  //Changes the border and background of the card when it is clicked
+  carlot.changeBorderAndBackground = function(event, orange) {
```

Comment:
Here you've set up the event to receive the event and a color (which is kind of oddly named as orange, since you could theoretically pass in any color), but you never send that argument, or use that argument, as you've hard-coded in the value "orange" at lines 12 and 19.

Also, while what you have works, the idea of the requirement to take a color as an argument was more along the lines that if you pass in a color name or value to this function, it changes the background of the card to that color.